modules/apps/layout/layout-seo-web/src/main/java/com/liferay/layout/seo/web/internal/configuration/FFOnlyTranslatedHreflangConfiguration.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.layout.seo.web.internal.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Alicia García
+ */
+@ExtendedObjectClassDefinition(generateUI = false)
+@Meta.OCD(
+	id = "com.liferay.layout.seo.web.internal.configuration.FFOnlyTranslatedHreflangConfiguration"
+)
+public interface FFOnlyTranslatedHreflangConfiguration {
+
+	@Meta.AD(deflt = "false", required = false)
+	public boolean onlyTranslatedHreflangEnabled();
```

Comment:
Call this method simply "enabled". We should also rename the FF as per my previous comments.

Reply:
On 

```
_ffSEOInlineFieldMapping = ConfigurableUtil.createConfigurable(FFSEOInlineFieldMapping.class, properties);
_ffOnlyTranslatedHreflangConfiguration =		ConfigurableUtil.createConfigurable(FFOnlyTranslatedHreflangConfiguration.class, properties);
```
On the properties the value "enabled" is present if one of the 2 FF is present activating both. because this the name must be different